### ISSUE TITLE
Replay command broken

### ISSUE THREAD

**Issue by Artforlfmj** [theo-bnts/custom-maker#3]
Getting err:

node:events:371
      throw er; // Unhandled 'error' event
      ^

Error: spawn NodeReplayBindings-win.exe ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (node:internal/child_process:288:12)
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn NodeReplayBindings-win.exe',
  path: 'NodeReplayBindings-win.exe',
  spawnargs: [
    'C:\\Users\\a_dep\\Documents\\GitHub\\custom-maker\\source\\assets\\temp\\1629904465851.replay',
    'Full'
  ]
}

Replay command is a priority?

**Comment by theo-bnts** [comment]
You should create a folder with this path : `source/assets/temp`

**Comment by theo-bnts** [comment]
> Replay command is a priority?
Yes and no, the command is currently broken and unusable but this has been the case for some time ...
